extensions/wikia/WikiaApi/WikiaApiQueryDomains.php

```diff
@@ -59,6 +59,23 @@ public function execute() {
 			if ( !empty($from) ) {
 				if ($from && is_int($from)) $this->addWhere('city_id >= '.intval($from));
 			}
+
+			if ( !empty( $from ) && !empty ( $to ) ) {
+				$settings = $this->getAllowedParams();
+
+				if ( $this->getMain()->canApiHighLimits() ) {
+					$maxLimit = $settings['limit'][ApiBase::PARAM_MAX2];
+				} else {
+					$maxLimit = $settings['limit'][ApiBase::PARAM_MAX];
+				}
+
+				$requested = 1 + $to - $from;
+				if ( $requested > $maxLimit ) {
+					$this->setWarning( "Count of results must not be over $maxLimit (set to $requested)" );
+				}
+
+				$limit = min( $requested, $maxLimit );
```

Comment:
Do we want `$limit` to be set only when `$from` and `$to` are not empty? What if any of those is not set? No limit at all?